site/blog/2022-11-04.mdx

```diff
@@ -0,0 +1,123 @@
+
+In this post, we're going to discover an IaSQL module that's built to make deploying to ECS, simplified. Most of the details for deploying a container to ECS are the same (load balancers, security groups, IAM roles, etc), and we have created the `ecs_simplified` module for you so that you can give it your codebase and get your app deployed to ECS **in just 11 minutes**. All of those needed resources are going to be created automatically in your AWS account, and you'll have full access over them either through the AWS dashboard, or through your IaSQL database.
+
+## ** High-level vs Low-level IaSQL Modules **
+In IaSQL, you are able to do cloud actions both on a high-level and a low-level abstraction layer. It means that you can go with both of these two approaches:
+1. Deploy the folder `examples/ecs-fargate/prisma/app` of the repo `https://github.com/iasql/iasql-engine/` to ECS on my AWS account
+2. Create a security group named `allow-all` attached to the VPC with ID `vpc-f56cff91`. Then add a rule to it allowing all incoming traffic on port `80` from `0.0.0.0/0`.
+
+The first statement, is described in a high-level abstraction layer. The higher the level, the less detail. So it doesn't say anything about the underlying steps that should be taken in order for the statement to be executed. But the second statement is giving all the details necessary to create a new security group.
+
+The flexibility of IaSQL– which makes it really awesome– is that you can create both the high-level and the low-level modules for it. Both of the above statements are already possible using IaSQL modules. The first one using a module named `aws_ecs_simplified` and the latter using the `aws_security_group` module. Let's take a peek on the actual implementations really quick before we continue our main topic.
+
+```sql title="Deploy the folder 'examples/ecs-fargate/prisma/app' of the 'iasql-engine' repo to ECS on my AWS account"
+-- Install the "high-level" aws_ecs_simplified IaSQL module
+SELECT iasql_install('aws_ecs_simplified');
+-- Create the load balancer, ECS service, task definition, etc. All ready afterwards for us to build and push to deploy the app.
+INSERT INTO ecs_simplified (app_name, app_port, image_tag, public_ip) VALUES ('simple-express', 8088, 'latest', true);
+SELECT iasql_apply();
+-- Use the "high-level" ecr_build function to build the folder "examples/ecs-fargate/prisma/app" from "iasql-engine" Github repo and push it to ECR
+SELECT ecr_build(
+  'https://github.com/iasql/iasql-engine/',
+  (SELECT id FROM repository WHERE repository_name = 'simple-express-repository')::varchar(255),
+  './examples/ecs-fargate/prisma/app',
+  'ghp_XXX', -- replace your github personal access token here
+  'main'
```

Comment:
It may make sense to put comments after each of these lines explaining what the arguments are. I know this is our `main` branch, but the reader may not know that and if they're still using `master` as the primary branch name, they might copy-paste this string as-is and not understand why it's failing.